scripts/test-dssspell-forge.sh

```diff
@@ -22,12 +23,16 @@ export FOUNDRY_OPTIMIZER=false
 export FOUNDRY_OPTIMIZER_RUNS=200
 export FOUNDRY_ROOT_CHAINID=5
 
-if [[ -z "$MATCH" && -z "$BLOCK" ]]; then
-    forge test --fork-url "$ETH_RPC_URL"
-elif [[ -z "$BLOCK" ]]; then
-    forge test --fork-url "$ETH_RPC_URL" --match-test "$MATCH" -vvv
-elif [[ -z "$MATCH" ]]; then
-    forge test --fork-url "$ETH_RPC_URL" --fork-block-number "$BLOCK"
-else
-    forge test --fork-url "$ETH_RPC_URL" --match-test "$MATCH" --fork-block-number "$BLOCK" -vvv
+TEST_ARGS=''
```

Comment:
Do you want to use the same `local` keyword as in the [mainnet PR](https://github.com/makerdao/spells-mainnet/pull/358)?
```suggestion
local TEST_ARGS=''
```

Reply:
Already reverted it.